UnitTests/GitCommandsTests/Settings/FileSettingsCacheTests.cs

```diff
@@ -50,16 +50,6 @@ public void ctor_CanEnableFileWatcher_should_be_false_if_invalid_dir(string sett
             new MockFileSettingsCache(settingsFilePath, false).GetTestAccessor().CanEnableFileWatcher.Should().BeFalse();
         }
 
-        [TestCase(null)]
-        [TestCase("")]
-        [TestCase("C:\\" + "\t")]
-        public void SaveImpl_should_throw_if_invalid_path(string settingsFilePath)
-        {
-            var cache = new MockFileSettingsCache(settingsFilePath, false).GetTestAccessor();
-            cache.SetLastModificationDate(DateTime.Now);
-            ((Action)(() => cache.SaveImpl())).Should().Throw<Exception>();
-        }
-
```

Comment:
I'd like to keep this test. Perhaps you can borrow the class `SimpleDialog` from #6348. It might be a good first part of the pending rework of that PR. Thoughts, @RussKie?

Reply:
I won't say "no" to more tests :)

Reply:
> I won't say "no" to more tests :)

That's no surprise :)
My question was more about tests with `MessageBox.Show`. #6348 introduces the class `SimpleDialog` with abstractions.

Reply:
I propose we decorate the test with `Ignore` attribute. 
Introduction of MessageBox abstractions should be done seprately

Reply:
Set to Ignore()
Please stop auto-merge if you believe this should be handled differently